internal/cmd/xdstest.go

```diff
@@ -0,0 +1,243 @@
+package cmd
+
+import (
+	"net"
+	"time"
+
+	"github.com/envoyproxy/gateway/internal/ir"
+	"github.com/envoyproxy/gateway/internal/log"
+	"github.com/envoyproxy/gateway/internal/xds/cache"
+	"github.com/envoyproxy/gateway/internal/xds/translator"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+
+	envoy_service_cluster_v3 "github.com/envoyproxy/go-control-plane/envoy/service/cluster/v3"
+	envoy_service_discovery_v3 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
+	envoy_service_endpoint_v3 "github.com/envoyproxy/go-control-plane/envoy/service/endpoint/v3"
+	envoy_service_listener_v3 "github.com/envoyproxy/go-control-plane/envoy/service/listener/v3"
+	envoy_service_route_v3 "github.com/envoyproxy/go-control-plane/envoy/service/route/v3"
+	envoy_service_runtime_v3 "github.com/envoyproxy/go-control-plane/envoy/service/runtime/v3"
+	envoy_service_secret_v3 "github.com/envoyproxy/go-control-plane/envoy/service/secret/v3"
+	envoy_server_v3 "github.com/envoyproxy/go-control-plane/pkg/server/v3"
```

Comment:
FWIW, I've found it mildly helpful to my sanity to name the imports to distinguish between Envoy's API (`github.com/envoyproxy/go-control-plane/envoy/…`) and the go-contro-plane's runtime (`github.com/envoyproxy/go-control-plane/pkg/…`).

Reply:
Yeah, this is a good idea, I'll update.